app/controllers/posts_controller.rb

```diff
@@ -1,8 +1,12 @@
 class PostsController < ApplicationController
-  load_and_authorize_resource
   def index
     @user = User.find(params[:user_id])
     @posts = @user.posts.includes(:comments, comments: :user)
+
+    respond_to do |format|
+      format.html # index.html.erb
+      format.json { render json: @posts }
+    end
   end
```

Comment:
- [ ] You should add an API endpoint that returns all posts in JSON format, however, I'm getting the HTML template instead of the expected response to the request (list of all posts). Please address this 👍🏻 

![image](https://github.com/anderCM/blog-app/assets/79058364/17489c67-c300-4861-ab69-b9d38a91103d)

Reply:
We're using the respond_to method in our controller which handles HTML and JSON requests. Need to append the request with .json or change it to JSON in your API client.

Reply:
@BenMukebo What my partner means is that we are managin both (html and json) to get the response, so please there are 2 options to get the data in `json` format using an external client.

1.  As my partner mentioned you could make a `GET` request to `http://localhost:3000/users/2/posts.json` using the `.json` extension at the end.
2. Make a `GET` request to `http://localhost:3000/users/2/posts` (without `.json` extension) but in the headers add `Accept: application/json` so the method could validate what are you trying to get like this example.
![image](https://github.com/anderCM/blog-app/assets/77707836/0a3d013c-e342-414d-83bd-1becf1657a5b)

Reply:
Hello @anderCM, @lucascomamala, Thank you for this information, please also provide more information from the README on how the reviewer can run or test your application based on the project requirements. ✍🏻 It can help avoid confisiion and save time trying to understand the logic behind the code